docs/using-tendermint.rst

```diff
@@ -246,13 +246,17 @@ conflicting messages.
 Note also that the ``pub_key`` (the public key) in the
 ``priv_validator.json`` is also present in the ``genesis.json``.
 
-The genesis file contains the list of public keys which may participate
-in the consensus, and their corresponding voting power. Greater than 2/3
-of the voting power must be active (ie. the corresponding private keys
-must be producing signatures) for the consensus to make progress. In our
-case, the genesis file contains the public key of our
-``priv_validator.json``, so a tendermint node started with the default
-root directory will be able to make new blocks, as we've already seen.
+The genesis file contains the list of public keys which may participate in the
+consensus, and their corresponding voting power. Greater than 2/3 of the voting
+power must be active (ie. the corresponding private keys must be producing
+signatures) for the consensus to make progress. Voting power range:
+-9223372036854775808 through 9223372036854775807. Because of how the current
+proposer selection algorithm works , we do not recommend having voting powers
+greater than 1000000000000 (see `Proposals section of Byzantine Consensus
+Algorithm <./specification/byzantine-consensus-algorithm.html#proposals>`__ for
+details). In our case, the genesis file contains the public key of our
+``priv_validator.json``, so a tendermint node started with the default root
+directory will be able to make new blocks, as we've already seen.
```

Comment:
`to make progress. The voting power range is: -922 ... 807.` rather than `to make progress. Voting power range: -922 ... 807.`, which isn't a sentence.

there's a extra space typo on L254: `proposer selection algorithm works , we`